var/spack/repos/builtin/packages/parallelio/package.py

```diff
@@ -29,24 +29,26 @@ class Parallelio(CMakePackage):
     variant(
         "fortran", default=True, description="enable fortran interface (requires netcdf fortran)"
     )
+    variant("mpi", default=True, description="Use mpi to build, otherwise use mpi-serial")
 
-    depends_on("mpi")
-    depends_on("netcdf-c +mpi", type="link")
+    depends_on("mpi", when="+mpi")
+    depends_on("mpi-serial", when="-mpi")
```

Comment:
We use `~` almost everywhere, can you change that throughout the recipe?
```suggestion
    depends_on("mpi-serial", when="~mpi")
```

Reply:
done